package.json

```diff
@@ -44,6 +44,9 @@
     "autoprefixer": "6.7.7",
     "axe-core": "2.0.7",
     "axe-webdriverjs": "0.4.0",
+    "babel-core": "6",
```

Comment:
@ukilon-okta I moved to use Yarn instead of package-lock in this PR, I updated the Readme file telling to use Yarn as cli :)